elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -168,6 +168,8 @@ elytron.syslog-audit-log.transport=The transport to use to connect to the syslog
 elytron.syslog-audit-log.format=The format to use to record the audit event.
 elytron.syslog-audit-log.host-name=The host name to embed withing all events sent to the remote syslog server.
 elytron.syslog-audit-log.ssl-context=The SSLContext to use to connect to the syslog server when SSL_TCP transport is used.
+elytron.syslog-audit-log.syslog-format=The RFC format to be used for describing the audit event, default value of RFC5424.
+elytron.syslog-audit-log.reconnect-attempts=The maximum number of reconnect attempts that should be made for sending the message with UDP, default value of 0 (no reconnect attempts).
```

Comment:
May need to double check this default. The comments in the analysis document mention that the default should be -1 but the description in the analysis document currently mentions 0.